index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang = "en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+    	<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<title>CSS Grid Framework - By Henry & Edward</title>
+		<link rel="stylesheet" href="./dist/css/style.css" />
+		<link rel="stylesheet" href="./theme.css" />
+		 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
+		 <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" rel="stylesheet">
+	</head>
+	<body>
+		<header>
+			<nav class = "navigation">
+				<div><i class="fab fa-pied-piper-alt fa-5x"></i></div>
+				<ul>
+					<li>Department</li>
+					<li>Today's Deal</li>
+					<li>Gift Cards</li>
+					<li>Help</li>
+					<li>Registry</li>
+					<li>Sell</li>
+				</ul>
+			</nav>
+		</header>
+		<div style class = "wrapper">
+			<div class="container">
+			<h4>Watches</h4>
+			<div class="row">
+				<div class = "col-sm-3 col-md-3 col-lg-3 col-xl-3">
+					<div class = "watches">
+						<img  src="./dist/images/Watch/watch1.jpg" alt = "watches">
+					</div>
+				</div>
+				<div class = "col-sm-3 col-md-3 col-lg-3 col-xl-3">
+					<div  class = "watches">
+						<img src="./dist/images/Watch/watch2.jpg" alt = "watches">
+					</div>
+				</div>
+				<div  class = "col-sm-3 col-md-3 col-lg-3 col-xl-3">
+					<div  class = "watches">
+						<img  src="./dist/images/Watch/watch3.jpg" alt = "watches">
+					</div>
+				</div>
+				<div class = "col-sm-3 col-md-3 col-lg-3 col-xl-3">
+					<div  class = "watches">
+						<img  src="./dist/images/Watch/watch4.jpg" alt = "watches">
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<div class="container">
+			<h4>Phones</h4>
+			<div class="row">
+			  	<div class = "col-sm-3 col-md-3 col-lg-3 col-xl-3">
+					<div class = "phones">
+						<img  src="./dist/images/Phones/phone1.jpg" alt = "phones">
+					</div>
+				</div>
+				<div class = "col-sm-3 col-md-3 col-lg-3 col-xl-3">
+					<div class = "phones">
+						<img class = "img-fluid" src="./dist/images/Phones/phone2.jpg" alt = "phones">
+					</div>
+				</div>
+				<div  class = "col-sm-3 col-md-3 col-lg-3 col-xl-3">
+					<div class = "phones">
+						<img  src="./dist/images/Phones/phone3.jpg" alt = "phones">
+					</div>
+				</div>
+				<div class = "col-sm-3 col-md-3 col-lg-3 col-xl-3">
+					<div class = "phones">
+						<img src="./dist/images/Phones/phone5.jpg" alt = "phones">
+					</div>
+				</div>
+			</div>
+		</div>
+
+		 <div class="container">
+			<h4>Video Games</h4>
+			<div class="row">
+				<div class = "col-sm-3 col-md-3 col-lg-3 col-xl-3">
```

Comment:
If you wanted it to span 4 columns on each screen size `col-sm-3` is all you need as it is from 576px minimum it would automatically cover medium, large and extra large screens. Only if the medium screen differs from the small screen and large from medium etc then you'd add those classes, otherwise, it is redundant to add `col-md-3`, `col-lg-3` etc. Please fix this for all occurences

With all that said I recommend you change the column sizes for small land medium screens. maybe the class could be like `class="col-sm-12 col-md-6 col-lg-3"` or `class="col-sm-6 col-lg-3"` etc. (Suggestion)